src/main/java/org/prebid/server/bidder/gumgum/GumgumBidder.java

```diff
@@ -108,6 +120,14 @@ private BidRequest createBidRequest(BidRequest bidRequest, List<BidderError> err
                 .build();
     }
 
+    @VisibleForTesting
+    protected String extractAdUnitId(Imp imp) {
+        if (imp.getExt() != null && imp.getExt().has("bidder") && imp.getExt().get("bidder").has("adunitid")) {
+            return imp.getExt().get("bidder").get("adunitid").asText();
+        }
```

Comment:
Add `adunitid` field to `ExtImpGumgum` and extract value from already parsed object.

Do as I said only if it is the right way because as I see from GO pr this field should be under `imp.ext.adunitid`